app/src/main/java/com/example/myapplicationwork/MainActivity.kt

```diff
@@ -27,4 +34,12 @@ class MainActivity : AppCompatActivity() , ClickListener {
         supportFragmentManager.popBackStack()
     }
 
+    override fun showDetail1() {
+        supportFragmentManager.beginTransaction()
+                .apply {
+                    replace(R.id.main_container, moviesDetailsFragment).addToBackStack(null)
```

Comment:
if you use apply receiver
you can write without dot(.)
I mean like so 
`replace(R.id.main_container, moviesDetailsFragment)
addToBackStack(null)
commit()`

but use **apply** and also use. it's nonsense